### PR TITLE
Update Trivy

### PR DESCRIPTION
**Pull request by itaysk**
was asked to create an new PR instead of https://github.com/cncf/landscape/pull/3898

### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [x] Are you only including a `repo_url` if your project is 100% open source? If so, you need to pick the single best GitHub repository for your project, not a GitHub organization.
* [x] Is your project closed source or, if it is open source, does your project have at least 300 GitHub stars?
* [x] Have you picked the single best (existing) category for your project?
* [x] Does it follow the other guidelines from the [new entries](https://github.com/cncf/landscape#new-entries) section?
* [x] Have you added your SVG to `hosted_logos` and referenced it there?
* [x] Does your logo clearly state the name of the project/product and follow the other [guidelines](https://github.com/cncf/landscape#new-entries)?
* [x] Does your project/product name match the text on the logo?
* [x] Have you verified that the Crunchbase data for your organization is correct (including headquarters and LinkedIn)?

### CODE REVIEW

File: landscape.yml [hunk 1]
```diff
@@ -2141,20 +2141,21 @@ landscape:
...
+            description: >-
+              Trivy is an all-in-one security scanner that beings vulnerability, SBOM, misconfigurations, leaked secrets, and license scanning together, across container images, code repositories, Kubernetes clusters and more. 
+            homepage_url: https://trivy.dev
             repo_url: https://github.com/aquasecurity/trivy
             logo: trivy.svg
             crunchbase: https://www.crunchbase.com/organization/aquasecurity
-          - item:
-            name: Trivy-Operator
+            twitter: https://x.com/aquatrivy
+           - item:
```
Comment (castrojo) [inline landscape.yml:2151]:
```suggestion
          - item:
```

**Review by castrojo — CHANGES_REQUESTED** [review]

File: landscape.yml (hunk 1 above)
Comment (itaysk) [reply landscape.yml]: thanks

**Comment by itaysk** [comment]
@castrojo I think this is ready? BTW, I'm not sure why trivy-operator was added separately from trivy (it wasn't us who added it). I wouldn't mind removing trivy-operator altogether since IMO it's a module of Trivy

**Comment by caniszczyk** [comment]
@itaysk if you want to remove the operator please send an updated PR, happy to if this basically is the same as your PR here

**Review by castrojo — APPROVED** [review]

**Comment by itaysk** [comment]
@caniszczyk should I wait for this to be merged and then send another PR or should I just remove it within this PR?

**Comment by castrojo** [comment]
@itaysk I'll merge this now, send one to remove it and tag me in it and i'll merge it asap, thanks!

**Comment by itaysk** [comment]
thanks @castrojo , I've created https://github.com/cncf/landscape/pull/4126

P.S I said earlier that we (Trivy) didn't add it but I checked and indeed it was added by our team member. Still, I don't see a special reason to have a separate entry for it.

**Pull request merged**